api/vudl.ini.dist

```diff
@@ -194,3 +194,118 @@ DOC = "vudl-system:DOCData"
 Audio = "vudl-system:AudioData"
 XLS = "vudl-system:XLSData"
 Text = "vudl-system:TextData"
+
+# AudioData
+
+[datastreams.WAV.mimetype]
+allowedType = "audio"
+allowedSubtypes = "wav,x-wav"
+
+[datastreams.MP3.mimetype]
+allowedType = "audio"
+allowedSubtypes = "mpeg3,x-mpeg-3,mpeg"
+
+[datastreams.MP3.mimetype]
+allowedType = "audio"
+allowedSubtypes = "ogg"
+
+# CollectionModel
+
+[datastreams.MEMBER-QUERY.mimetype]
```

Comment:
I don't think we need to keep configuration for deprecated/legacy datastreams in the default configuration. Things like MEMBER-QUERY, MEMBER-LIST-RAW, STRUCTMAP, etc., etc. will never be added to the system going forward, so I don't think we need to specify legal types. Hopefully that will significantly reduce the size of this configuration.

Reply:
Made change.